test/WebpackConfig.js

```diff
@@ -940,24 +940,15 @@ describe('WebpackConfig object', () => {
         it('Adds new externals', () => {
             const config = createConfig();
 
-            expect(config.externals).to.deep.equals({});
+            expect(config.externals).to.deep.equals([]);
 
             config.addExternals({ 'jquery': 'jQuery', 'react': 'react' });
             config.addExternals({ 'lodash': 'lodash' });
```

Comment:
Can you update this test by adding, for example, another `addExternal()` that passes a regex?